services/motion/builtin/builtin_test.go

```diff
@@ -859,8 +861,10 @@ func TestCheckPlan(t *testing.T) {
 		worldState, err := referenceframe.NewWorldState(gifs, nil)
 		test.That(t, err, test.ShouldBeNil)
 
-		err = motionplan.CheckPlan(moveRequest.kinematicBase.Kinematics(), plan, worldState, newFS, startPose, inputs, errorState, logger)
-		test.That(t, err, test.ShouldNotBeNil)
+		collisionPose, err := motionplan.CheckPlan(moveRequest.kinematicBase.Kinematics(), plan, worldState, newFS,
+			startPose, inputs, errorState, logger)
+		test.That(t, err, test.ShouldBeNil)
```

Comment:
This test was ensuring a failure occurred. Why does a failure no longer occur?

Reply:
Thanks for catching this, I missed changing the test back to original values due to it being skipped.